### PR TITLE
feat: Show ae amount for add_vote Smart Contract call

### LINKED ISSUE #685: Show ae amount for add_vote Smart Contract call
### Please describe the problem that your request should resolve.

the transaction Smart Contract Called add_vote includes considerable ae as amount, would be nice if this could be shown in the activity overview as well 

### Describe the solution you'd like
Show ae amount

### Is it already possible to achieve the same outcome in any other way? If so, how?
No

### Additional context
https://testnet.aescan.io/accounts/ak_3CoDS4wS7N1bXZJWUgmRXNez45AbMVFG5gsVpHx9kRsWcrhp9?type=activities
https://testnet.aescan.io/transactions/th_H89SCRqTNjXP6s4saD5Ae4a37VQb3bbR5hy9FDqugXA8kat29

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #685

## Demo
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] show amount label to add_vote activity

### CODE REVIEW

File: src/components/AccountActivityDataCellContractCallTxEvent.vue [hunk 1]
```diff
@@ -6,11 +6,17 @@
   <app-chip size="sm">
     {{ activity.payload.tx.function }}
   </app-chip>
+  <app-chip
+    v-if="activity.payload.tx.function === 'add_vote'"
```
Comment (thepiwo) [inline src/components/AccountActivityDataCellContractCallTxEvent.vue:10]: why do we only want to show this for this function? I'd propose to show it wherever amount !== 0

Comment (janmichek) [reply src/components/AccountActivityDataCellContractCallTxEvent.vue]: Yeah, makes sense. Thanks for the suggestion. I checked it and didn't find any issue with it. Improved.

**Review by thepiwo — APPROVED** [review]

**Review by michele-franchi — APPROVED** [review]
Looks good

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Pull request merged**